src/utils.js

```diff
@@ -20,44 +19,34 @@ const exec = async (command) => {
   });
 };
 
-const is_svg = async (opts) => {
-  const { path, buffer } = opts;
-
-  if (path && PATH.extname(path).toLowerCase() === '.svg') return true;
-
-  const svg_candidate = path
-    ? await fs.promises.readFile(path)
-    : buffer.toString('utf-8');
-
-  if (isSvg(svg_candidate)) return true;
-
-  return false;
-};
-
 const mime_type = async (opts) => {
-  const { path, buffer } = opts;
-
-  try {
-    if (await is_svg(opts)) return 'image/svg+xml';
-
-    let mime;
-    if (path) ({ mime } = await FileType.fromFile(path));
-    else ({ mime } = await FileType.fromBuffer(buffer));
-
-    return mime;
-  } catch (err) {
-    throw new Error(
-      `Failed guessing mime type of ${path ? `file ${path}` : `buffer`}`
-    );
-  }
+  return new Promise((resolve, reject) => {
```

Comment:
Why `return new Promise` instead of taking advantage of `async` to `throw` and `return` instead of calling the `reject` and `resolve` functions?

Reply:
The library does not support promises. When that happen you have to promisify the function.

Reply:
I see, what a pity!